scripts/vim-patch.sh

```diff
@@ -326,12 +326,14 @@ stage_patch() {
   return $ret
 }
 
-hub_pr() {
-  hub pull-request -m "$1"
+gh_pr() {
+  gh pr create --title "$1" --body "$2"
 }
 
 git_hub_pr() {
-  git hub pull new -m "$1"
+  local pr_message
+  pr_message="$(printf '%s\n\n%s\n' "$1" "$2")"
+  git hub pull new -m "${pr_message}"
```

Comment:
it is still using hub.

Reply:
No, [git-hub](https://github.com/sociomantic-tsunami/git-hub/) is a separate project.  Only hub should be replaced by gh.

Reply:
While at it we could ditch git-hub too, depends on @janlazo that uses the script the most I guess.